tfsdk/config_test.go

```diff
@@ -0,0 +1,222 @@
+package tfsdk
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	testtypes "github.com/hashicorp/terraform-plugin-framework/internal/testing/types"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+func TestConfigGet(t *testing.T) {
+	testConfig := Config{
+		Raw: tftypes.NewValue(tftypes.Object{
+			AttributeTypes: map[string]tftypes.Type{
+				"name": tftypes.String,
+			},
+		}, map[string]tftypes.Value{
+			"name": tftypes.NewValue(tftypes.String, "namevalue"),
+		}),
+		Schema: Schema{
+			Attributes: map[string]Attribute{
+				"name": {
+					Type:     types.StringType,
+					Required: true,
+				},
+			},
+		},
+	}
+
+	type testConfigGetData struct {
+		Name types.String `tfsdk:"name"`
+	}
+
+	var val testConfigGetData
+
+	diags := testConfig.Get(context.Background(), &val)
+
+	if diagsHasErrors(diags) {
+		t.Fatalf("unexpected error: %s", diagsString(diags))
+	}
+
+	expected := testConfigGetData{
+		Name: types.String{Value: "namevalue"},
+	}
+
+	if diff := cmp.Diff(val, expected); diff != "" {
+		t.Errorf("unexpected diff (+wanted, -got): %s", diff)
+	}
+}
+
+func TestConfigGet_AttrTypeWithValidate_Error(t *testing.T) {
```

Comment:
I think all these tests need the `t.Parallel` call in them so they'll run in parallel.

Reply:
Will update these to include `t.Parallel()` regardless of TDTing or not.

Reply:
`t.Parallel()`-ized all the `Config`, `Plan`, and `State` testing (including existing tests) in c8cef290dea05aa5330a6645be43bf2b5350e23a